wporg-gp-translation-events.php

```diff
@@ -134,6 +134,32 @@ function save_event_meta_boxes( $post_id ) {
 	}
 }
 
+/**
+ * Generate a unique event slug.
+ *
+ * @param  string $title The event title.
+ * @param  int    $post_id The event post ID.
+ * @return string The unique event slug.
+ */
+function generate_unique_event_slug( $title, $post_id = 0 ) {
```

Comment:
I noticed that there's a [`wp_unique_post_slug()` function](https://developer.wordpress.org/reference/functions/wp_unique_post_slug/). Would it make sense to use it here?

Reply:
Thank you for this!  So, the `generate_unique_event_slug` function will no longer be needed. I'll go ahead and close this PR.